frontend/src/Components/ProductDescription/ProductDescription.jsx

```diff
@@ -0,0 +1,138 @@
+        hideToast(dispatch)
+        }
+    }
+
+    const productAddToWishlistHandler = async () => {
+        if(!auth.isLoggedIn) {
+            dispatch({type:"TOGGLE_TOAST",payload:"Login Toast"});
+            hideToast(dispatch, 3000);
+        }
+        else {
+        if(isInWishlist(state.wishlist,product._id)){
+        dispatch({type:"TOGGLE_TOAST",payload:"removing from wishlist...", value: true});
+        // const {response} = await RestApiCalls("DELETE", `${BACKEND}/${auth.user._id}/wishlist/${_id}`)
+        const {data : {response, success}} = await axios.delete(`${BACKEND}/${auth.user._id}/wishlist/${product._id}`);
+        if(success) {
+        dispatch({type: "SET_WISHLIST", payload: response.wishlistItems});
+        dispatch({type:"TOGGLE_TOAST",payload:"1 item removed from wishlist"});
+        hideToast(dispatch)
+        }
+        }
+        else {
+        dispatch({type:"TOGGLE_TOAST",payload:"adding to wishlist...", value: true});
+        const {response} = await RestApiCalls("POST", `${BACKEND}/${auth.user._id}/wishlist/${product._id}`)
+        dispatch({type: "SET_WISHLIST", payload: response.wishlistItems});
+        dispatch({type:"TOGGLE_TOAST",payload:"1 item added to wishlist"});
+        hideToast(dispatch)
+        }
+        }
+    }
+    return product && 
+        (<>
+        <div className = "product-description">
+            <div className = "product-description__left">
+                <img src = {product.image} alt = "" className = "responsive-img"></img>
+            </div>
+            <div className = "product-description__right">
+                <div className = "product-description__heading">
+                    <h1 className = "brand-name rm">
+                        {product.brandName}
+                    </h1>
+                    <div className = "description">
+                        {product.description}
+                    </div>
+                    <StarRatingTool rating = { product.rating } />
+
+                </div>
+                <div className = "product-description__price">
+                    <div className = "price">
+                    <h3 className = "rm"><strong>Rs. {product.price.split(".")[0]} </strong></h3>
+                        {product.discountByPercentage !== 0 && <h3 className="rm light strikethrough">Rs. {calculateOriginalPrice(product.price,product.discountByPercentage)} </h3>}
+                        {product.discountByPercentage !== 0 && <h3 className="rm discount">({product.discountByPercentage}% OFF)</h3>}
+                    </div>
+                    <small className = "green-txt">inclusive of all taxes</small>
+                    <div className = "product-description__buttons flex">
+                        {product.outOfStock ?  
+                            <button className = "btn btn-primary flex disabled" disabled>
+                            <ShoppingCartIcon style = {{marginRight: "0.5rem"}}/> OUT OF STOCK
```

Comment:
Avoid inline styles as much as possible.

Reply:
Since it's a material Ui Component used this, will use makeStyles()